### PR TITLE
Tenant support

### PR DESCRIPTION
**Pull request by code-wizard**
Add multi-schema support and Celery 5 support

### CODE REVIEW

**Comment by cclauss** [comment]
Please fix the git conflicts.

File: celery_haystack/conf.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class CeleryHaystack(AppConf):
     COMMAND_BATCH_SIZE = None
     #: The max age of items used by the CeleryHaystackUpdateIndex task
     COMMAND_AGE = None
-    #: Whether to remove items from the index that aren't in the DB anymore
+    #: Wehther to remove items from the index that aren't in the DB anymore
```
Comment (cclauss) [inline celery_haystack/conf.py:28]: Please undo this change.  The spelling is correct.

File: celery_haystack/signals.py [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 from haystack.signals import BaseSignalProcessor
 from haystack.exceptions import NotHandled
-
```
Comment (cclauss) [inline celery_haystack/signals.py:5]: Please do not undo the work of [isort](https://github.com/PyCQA/isort#readme).

File: celery_haystack/tasks.py [hunk 3]
```diff
@@ -151,5 +156,16 @@ def run(self, apps=None, **kwargs):
...
         logger.info("Finishing update index")
+
+
+@shared_task()
+def haystack_signal(action, identifier, **kwargs):
+    task = CeleryHaystackSignalHandler()
+    task.run(action, identifier, **kwargs)
+
+
+@shared_task()
```
Comment (cclauss) [inline celery_haystack/tasks.py:169]: Please fix ⛔ below.

File: celery_haystack/utils.py [hunk 4]
```diff
@@ -23,7 +23,8 @@ def get_update_task(task_path=None):
     except AttributeError:
         raise ImproperlyConfigured('Module "%s" does not define a "%s" '
                                    'class.' % (module, attr))
-    return Task()
+    return Task
```
Comment (cclauss) [inline celery_haystack/utils.py:26]: Do we want to return a class or an instance of that class?

**Comment by code-wizard** [comment]
Closing this PR it was supposed to be a PR for a forked version
